cmd/registry-experimental/cmd/generate/openapi.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2023 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generate
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/apigee/registry/cmd/registry/core"
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/pkg/connection"
+	"github.com/apigee/registry/rpc"
+	"github.com/apigee/registry/server/registry/names"
+	"github.com/spf13/cobra"
+)
+
+func openapiCommand(ctx context.Context) *cobra.Command {
+	var specID string
+	cmd := &cobra.Command{
+		Use:   "openapi",
+		Short: "Generate an OpenAPI spec from another specificaton format",
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Are we switching to RunE?

Reply:
Yes, I'd say that's being driven by testing (when I've need to test error handling, I've switched to `RunE`).